lib/samsam.test.js

```diff
@@ -4,12 +4,38 @@ var referee = require("referee");
 var assert = referee.assert;
 var refute = referee.refute;
 
-describe("SamSam", function () {
-    it("Should Always Be true", function () {
-        assert.equals(42, 42);
-    });
-
-    it("Should Always Be False", function () {
-        refute.equals(42, 12);
-    });
-});
+var samsam = require("./samsam");
+
+describe("isElement", function() {
+    it("Should pass if DOM element node", function() {
+        if (typeof document !== "undefined") {
```

Comment:
Con: `document` will always be undefined as we don't run the tests in the browser, So we need to find another way to test this, as the tests don't really go through it.

Reply:
There are different options, you could set up tests using Chromium like we do in [sinonjs/sinon](https://github.com/sinonjs/sinon), or use `jsdom` to make a synthetic DOM.

(It would probably be a LOT less effort to use `jsdom`)

Reply:
I've tried jsdom before, it defines `document`, but I can't figure out why `var div` [here](https://github.com/sinonjs/samsam/blob/master/lib/samsam.js#L18) is always false.

Reply:
I've added it to the last commit if you want to have a look.

Reply:
The secret that I forgot to tell you was to use [`jsdom-global`](https://github.com/rstacruz/jsdom-global) in the test file before loading `samsam.js`. Once you have that, then `div` won't always be false and the test passes